src/raftstore/store/config.rs

```diff
@@ -92,6 +92,8 @@ pub struct Config {
     pub abnormal_leader_missing_duration: ReadableDuration,
     pub peer_stale_state_check_interval: ReadableDuration,
 
+    pub leader_transfer_max_log_lag: u64,
```

Comment:
no need to add to the config example file?

Reply:
Let's hide it from config, since it is an advanced configuration and also a workaround.